MarsQACompetitionTaskNUnit/Pages/EducationPage.cs

```diff
@@ -0,0 +1,326 @@
+﻿using MarsQACompetitionTaskNUnit.Utilities;
+using OpenQA.Selenium;
+
+namespace MarsQACompetitionTaskNUnit.Pages
+{
+    public class EducationPage : CommonDriver
+    {
+       
+        public EducationPage(IWebDriver driver) : base(driver)
+        {
+            this.driver = driver;
```

Comment:
same comments apply

Reply:
Even though we are extending a common driver class, we still need the constructor in the EducationPage class to explicitly receive and use the IWebDriver instance. This ensures that each page object has the necessary driver to perform its operations correctly(please review Base Test)